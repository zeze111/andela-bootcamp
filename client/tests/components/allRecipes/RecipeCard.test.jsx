import React from 'react';

import RecipeCard from '../../../components/allRecipes/RecipeCard';
import store from '../../../store';

const props = {
  recipe: {
    id: 0,
    name: 'something'
  }
};

describe('Test for Recipe Card', () => {
  it('should render component', () => {
    const wrapper = shallow(<RecipeCard {...props} store={store}/> );
    expect(wrapper).toMatchSnapshot()
  });
});
