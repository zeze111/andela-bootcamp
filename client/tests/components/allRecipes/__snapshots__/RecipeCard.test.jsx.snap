// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Recipe Card should render component 1`] = `
<li>
  <div
    className="col s2 offset-1"
  >
    <div
      className="card hoverable grey lighten-4 home-cards"
    >
      <div
        className="card-image"
      >
        <img
          alt="recipe"
          className="responsive-img image-style"
          src="/images/noimg.png"
        />
        <div
          className="card-action"
        >
          <Link
            className="home-text-style"
            data-tip="something"
            href="/recipe/0"
            replace={false}
            to="/recipe/0"
          >
             
            something...
            <ReactTooltip
              insecure={true}
              resizeHide={true}
              wrapper="div"
            />
          </Link>
           
          <span
            className="home-text2"
          >
            <i
              className="material-icons home-icon2"
            >
              visibility
            </i>
          </span>
        </div>
      </div>
    </div>
  </div>
</li>
`;
